components/BUILD.yaml

```diff
@@ -174,3 +175,35 @@ scripts:
         mysql -e "$query" -u$DB_USERNAME -p$DB_PASSWORD -h 127.0.0.1 gitpod
       fi
       kill $PID || true
+
+  - name: make-user-admin
+    deps:
+      - components/service-waiter:app
+    script: |
+      export DB_USERNAME=root
+      export DB_PASSWORD=test
+      kubectl port-forward statefulset/mysql 3306 &
+      PID=$!
+      service-waiter database -t 10s
+      if [ $? -ne 0 ]; then
+        echo "could not connect to DB"
+        kill $PID || true
+        exit 1
+      fi
+      query="select name, rolesOrPermissions from d_b_user limit 10;"
+      result=$(mysql -e "$query" -u$DB_USERNAME -p$DB_PASSWORD -h 127.0.0.1 gitpod)
+
+      printf "\nCurrent context:           $(kubectx -c)"
+      printf "\nCurrent namespace:         $(kubens -c)\n"
+
+      printf "\nAvailable users (max 10):\n$result\n\n"
+      printf "Enter user to unblock (empty to abort): "
+      read user
```

Comment:
Leeway scripts also support arguments, i.e. instead of reading this from stdin you could have folks pass it in when calling the script.

```suggestion
      user=${user}
```

would result in
```
# fails
$ leeway run components:make-user-admin
FATA[0001] cannot run script with unresolved argument "${user}": use -Duser=value to set the argument

# works
$ leeway run components:make-user-admin -Duser=foobar
```

Reply:
Indeed, that would be a nice alternative. Thanks for the hint. However, that's an additional thing that everyone that would like to use this command has to remember (the `-Duser=foobar` syntax). That's why I decided to use the current implementation. But I am not dogmatic about it.